advanced_functionality/unity_ml_agents/src/train.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import numpy as np
+import os
+import tensorflow as tf
+from mlagents_envs.environment import UnityEnvironment
+import mlagents
+import subprocess
+import yaml
+import json
+
+def parse_args():
+    
+    parser = argparse.ArgumentParser()
+
+    # hyperparameters sent by the client are passed as command-line arguments to the script
+    parser.add_argument('--epochs', type=int, default=1)
+    parser.add_argument('--batch_size', type=int, default=64)
+    parser.add_argument('--learning_rate', type=float, default=0.1)
+    
+    # data directories
+    parser.add_argument('--train', type=str, default=os.environ.get('SM_CHANNEL_TRAIN'))
```

Comment:
A minor question, why RL setup need train/test data set location here?

Reply:
Using the train channel for a binary file, no need for test channel